bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/async/StrandMetadata.java

```diff
@@ -17,38 +17,36 @@
  */
 package io.ballerina.runtime.api.async;
 
-import io.ballerina.runtime.internal.scheduling.Strand;
-
 /**
- * Holds metadata of the @{@link Strand}.
+ * Holds metadata of a Ballerina strand.
```

Comment:
Shouldn't we need to provide the link to the `Strand` class here?

Reply:
`Strand` is a `runtime.internal` class. So, we should not expose it here.